scripts/generate-icons.py

```
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
+"""
+PERC-529: Generate Percolator app icons + splash screen
+Implements DESIGN-BRIEF-MOBILE-V2.md Section 1 spec.
+"""
+
+import os
+from PIL import Image, ImageDraw, ImageFilter, ImageFont
+
+FONT_PATH = "/tmp/JetBrainsMono-Bold-Italic.ttf"
```

Ruff (0.15.4)

[error] Probable insecure usage of temporary file or directory: "/tmp/JetBrainsMono-Bold-Italic.ttf"

(S108)